client/view/base/font-lato.css

```diff
@@ -0,0 +1,27 @@
+@font-face {
+  font-family: 'Lato';
+  font-weight: 400;
+  src: url(./font-lato/Lato-Regular.woff2) format('woff2');
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'Lato';
+  font-weight: 600;
+  src: url(./font-lato/Lato-Black.woff2) format('woff2');
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'Lato';
+  font-weight: 800;
+  src: url(./font-lato/Lato-Black.woff2) format('woff2');
+  font-display: swap;
+}
+
+@font-face {
+  font-family: 'Lato Fallback';
+  src: local(Arial);
+  size-adjust: 97.38%;
```

Comment:
Cool feature! 👍

Reply:
I recommend [this service](https://deploy-preview-15--upbeat-shirley-608546.netlify.app/perfect-ish-font-fallback/?font=Montserrat) if you work with fonts from Google Fonts.

There is no need to parse OpenType :)